tests/test_unifi_client.py

```
@@ -0,0 +1,343 @@
+
+    # Test get_network_config()
+    config = unifi_client.get_network_config("default")
+    assert isinstance(config, NetworkConfiguration)
+    assert config.name == "Default"
+    assert config.purpose == "corporate"
+    assert config.enabled is True
+    assert config.subnet == "192.168.1.0"
+
+
+@patch("requests.Session.request")
+def test_system_status(mock_request, unifi_client, mock_system_response):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_system_status`

Add return type annotation: `None`

(ANN201)